docs/045-employee-handbook-ca/benefits-and-holidays.md

```diff
@@ -19,7 +19,7 @@ For employees in Ontario, CivicActions Canada is considered closed on the follow
 | Christmas Day  | December 25th            |
 | Boxing Day     | December 26th            |
 
-If a holiday falls on a the weekend, the Ontario office observes on the preceding Friday for Saturday holidays and the following Monday for Sunday holidays.
+If a holiday falls on a the weekend, the Ontario office observes on the Monday for the holiday.
```

Comment:
@elizabethraley "observes on the Monday for the holiday" isn't clear to me? Should this be "observes the holiday on the following Monday"?